packages/plugin-ext/src/plugin/plugin-icon-path.ts

```diff
@@ -18,10 +18,11 @@ import * as path from 'path';
 import { URI } from './types-impl';
 import { IconUrl, PluginPackage } from '../common/plugin-protocol';
 import { Plugin } from '../common/plugin-api-rpc';
+import { ThemeIcon } from '@theia/monaco-editor-core/esm/vs/platform/theme/common/themeService';
 
-export type PluginIconPath = string | URI | {
-    light: string | URI,
-    dark: string | URI
+export type PluginIconPath = string | URI | ThemeIcon | {
+    light: string | URI | ThemeIcon,
```

Comment:
Since the ThemeIcon comes from a Theme, which is already light or dark, does it make sense the have a "light ThemeIcon" and a "dark ThemeIcon"?